homeassistant/components/opengarage/button.py

```diff
@@ -0,0 +1,89 @@
+"""OpenGarage button."""
+from __future__ import annotations
+
+from collections.abc import Callable
+from dataclasses import dataclass
+from typing import Any, cast
+
+import opengarage
+
+from homeassistant.components.button import (
+    ButtonDeviceClass,
+    ButtonEntity,
+    ButtonEntityDescription,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import EntityCategory
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import OpenGarageDataUpdateCoordinator
+from .const import DOMAIN
+from .entity import OpenGarageEntity
+
+
+@dataclass
+class OpenGarageButtonEntityDescriptionMixin:
+    """Mixin to describe a OpenGarage button entity."""
+
+    press_action: Callable[[opengarage.OpenGarage], Any]
+
+
+@dataclass
+class OpenGarageButtonEntityDescription(
+    ButtonEntityDescription, OpenGarageButtonEntityDescriptionMixin
+):
+    """OpenGarage Browser button description."""
+
+
+BUTTONS: tuple[OpenGarageButtonEntityDescription, ...] = (
+    OpenGarageButtonEntityDescription(
+        key="reboot_device",
+        translation_key="reboot_device",
```

Comment:
For certain platform you can add a device class and it will take the name of the device class. So if you remove the translation key here and from strings.json, the button will be called "Restart" (which is also in line with other integrations)